.github/workflows/separate_env_public_remove.yml

```diff
@@ -0,0 +1,32 @@
+name: Separate environment remove
+on:
+  workflow_dispatch:
+
+jobs:
+  separate-env-remove:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: get branch name
```

Comment:
Please re-think about using demo env name instead of branch

Reply:
Changed the logics to use input var ENV_NAME to be used instead of branch name.